docs/mkdocs/docs/api/macros/index.md

```diff
@@ -15,6 +15,7 @@ header. See also the [macro overview page](../../features/macros.md).
 
 ## Language support
 
+- [**JSON_DISABLE_ENUM_SERIALIZATION**](json_disable_enum_serialization.md) - switch off default parse/serialization functions for enums
```

Comment:
Doesn't this rather belong to the section `Type conversion` below?

Reply:
Yes, definitely, moved.

Reply:
Oh, I didn't see your comment. I was thinking `Serialization/deserialization macros` would be a better fit?

Reply:
Yes, see my comment below https://github.com/nlohmann/json/pull/3536#discussion_r898817703:

> I though so too, but have seen parse used more in the documentation. But it does makes sense, you parse json and then deserialize it into specified type.
>
>Changed it so it is serialization/deserialization and reflected this change in json_disable_enum_serialization.md also.